bgpd/bgp_route.c

```diff
@@ -7844,6 +7937,8 @@ DEFUN (show_ip_bgp_ipv4,
              |filter-list WORD\
              |community [<AA:NN|local-AS|no-advertise|no-export> [exact-match]]\
              |community-list <(1-500)|WORD> [exact-match]\
+             |large-community [<AA:BB:CC>...]\
```

Comment:
1. Makes no sense to put one item in mutual exclusion and is disallowed.
2. Variadics (...) are only allowed as the very last item in a command.

Both of these problems will result in a parse failure. If bgpd is not logging that to whatever its log facility is, that is a bug that needs to be fixed.

Break off the show command for large communities in its own command.